### PR TITLE
Added start/end padding to SelectItemQuizView

### PR DESCRIPTION
**Pull request by appoll**
Considered that 8dp padding would be nice to have. See attached screenshots for before & after.

device-2015-08-11-124437
device-2015-08-16-155508

### CODE REVIEW

**Comment by appoll** [comment]
I signed it!

File: app/src/main/res/layout/item_answer_start.xml [hunk 1]
```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (keyboardsurfer) [inline app/src/main/res/layout/item_answer_start.xml]: Please fix the formatting for this file's license header.

**Comment by keyboardsurfer** [comment]
Thank you. Much appreciated.

**Comment by keyboardsurfer** [comment]
Cherry picked onto master.

**Pull request closed without merging**